app/assets/javascripts/darkswarm/controllers/registration/registration_form_controller.js.coffee

```diff
@@ -9,7 +9,7 @@ angular.module('Darkswarm').controller "RegistrationFormCtrl", ($scope, Registra
   $scope.create = (form) ->
     if ($scope.valid(form)) 
       $scope.disableButton()
-      EnterpriseRegistrationService.create().then(() ->
+      EnterpriseRegistrationService.create($scope.enableButton).then(() ->
         $scope.enableButton()
       )
     end
```

Comment:
I am not sure what's this is doing here, could you remove it ? Thanks !

Reply:
Hello @rioug , as you advised me to do, I added the `$scope.enableButton` callback
Cf. https://github.com/openfoodfoundation/openfoodnetwork/pull/11177#discussion_r1257621636
Your suggestion of the 4 lines for a create method, that exists already in `registration_form_controller.js.coffee`, so I just added the callback.

Reply:
I tried to comment on the `end` but it picked up the change above, sorry for the confusion ! Could you remove that `end` ?

Reply:
No problemo. I have removed the `end`